playback-sdk-android/src/main/java/com/streamamg/player/ui/BackgroundPlaybackService.kt

```diff
@@ -0,0 +1,111 @@
+package com.streamamg.player.ui
+
+import android.app.Notification
+import android.app.Service
+import android.content.Context
+import android.content.Intent
+import android.os.Binder
+import android.os.IBinder
+import androidx.core.app.NotificationCompat
+import com.bitmovin.player.api.Player
+import com.bitmovin.player.api.PlayerConfig
+import com.bitmovin.player.api.ui.notification.CustomActionReceiver
+import com.bitmovin.player.api.ui.notification.NotificationListener
+import com.bitmovin.player.ui.notification.DefaultMediaDescriptor
+import com.bitmovin.player.ui.notification.PlayerNotificationManager
+import com.streamamg.PlaybackSDKManager
+import com.streamamg.playback_sdk_android.R
+
+private const val NOTIFICATION_CHANNEL_ID = "com.streamamg.playback"
```

Comment:
```suggestion
companion object {
        private const val NOTIFICATION_CHANNEL_ID = "playback_channel"
        private const val NOTIFICATION_ID = 1
        private const val EMPTY_CHANNEL_DESCRIPTION = ""
    }
```

Can we move the constants to companion object ?

Reply:
@zachtom sure, any reason for that change? I've just followed the bitmovin example here:
https://github.com/bitmovin/bitmovin-player-android-samples/blob/6c6125fe70e7845f933be63abfcb286236ea8f77/BackgroundPlaybackKotlin/src/main/java/com/bitmovin/player/samples/playback/background/BackgroundPlaybackService.kt#L18

Reply:
Both way it works, just bit more neater thats all.

Reply:
Thanks @zachtom fixed!